File: uri/1074/1074.cpp
#include <cstdio>

int main(){
    int q, n;
    scanf("%d", &q);
    while(q--){
        scanf("%d", &n);
        if (n == 0){
            printf("NULL\n");
            continue;
        }

        if (n % 2 == 0)
            printf("EVEN ");
        else
            printf("ODD ");

        if (n < 0)
            printf("NEGATIVE\n");
        else
            printf("POSITIVE\n");
    }

    return 0;
}

File: solutions/beecrowd/1074/1074.py
n = int(input())

for _ in range(n):
    number = int(input())

    if number == 0:
        print('NULL')
        continue

    if number % 2 == 0:
        print('EVEN ', end='')
    else:
        print('ODD ', end='')

    if number > 0:
        print('POSITIVE')
    else:
        print('NEGATIVE')
